tools/universe_builder.py

```diff
@@ -20,6 +20,8 @@
 _jre_jce_unlimited_url = 'https://downloads.mesosphere.com/java/jre-8u131-linux-x64-jce-unlimited.tar.gz'
 _libmesos_bundle_url = 'https://downloads.mesosphere.io/libmesos-bundle/libmesos-bundle-1.10-1.4-63e0814.tar.gz'
 
+_docs_root = "https://docs.mesosphere.com"
```

Comment:
In the future we'll probably want this overridable for partners.

Reply:
Yes, definitely. We can already override the final templating values by setting a `TEMPLATE_DOCUMENTATION_PATH` environment variable (in keeping with the other template variables), but this may not be sufficient.